agreement_legal/data/demo.xml

```diff
@@ -0,0 +1,148 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+     © 2019 Ygol Internetwork (yves@ygol.com)
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+-->
+
+<odoo noupdate="1">
+
+    <function name="write" model="ir.model.data">
+        <function name="search" model="ir.model.data">
+            <value
+                eval="[('module', '=', 'agreement'), ('name', '=', 'market1')]"
+            />
+        </function>
+
+        <value eval="{'noupdate': False}" />
+    </function>
```

Comment:
@ygol I am not sure to get the purpose of this block. Can you explain please?

Reply:
It is not needed indeed. tnx